src/slots/Slot.js

```diff
@@ -94,7 +76,7 @@ class Slot {
       name: Joi.string().required(),
       displayType: Joi.string().allow(null).required(),
       dataStreams: Joi.array().items(DataStream.SCHEMA).required(),
-      unit: Joi.string().allow(...this.ALL_UNIT_VALUES).required(),
```

Comment:
could either leave in so when we add the specific unit validations its straightforward or yank for now understanding its not doing anything